[10.10-maintenance] gulpfile.js - remove gconf2 requirement

nerdCopter opened the pull request:
- uncertain if this is acceptable/proper for all distributions.
- needs testing
- built and installed fine on Debian 12 that already has both gconf2 and dconf installed.
- tried requirement `DConf` as well with no noticable difference.
- need for this is debatable.

nerdCopter commented:
@pitts-mo, @hegelocampus, welcome to test the above builds (https://github.com/betaflight/betaflight-configurator/pull/4114#issuecomment-2256820580).

SuperPi911 commented:
Still needs libgconf-2-4 , the deb package is not installable in linuxmint 22/ ubuntu 24.04

nerdCopter commented:
okay, shallow investigation: seems to be a `gulp-debian` dependency which is alrady latest version 0.3.2 at  yrs old.
will investigate further later, probably less likely to fix as PWA https://app.betaflight.com is the now and future.

nerdCopter commented:
added `dconf` as a dependency and updated `nw-builder` to version `3.8.6` which is the last version that does not require additional changes.
(technically 4.8.1 is the latest nw-builder version, but many modifications would be required to build.)

nerdCopter commented:
Tested successful on Fedora VM: (requires usual udev rules, etc.)
![2024-08-13_101050](https://github.com/user-attachments/assets/80011996-f663-41c7-8917-11972fc2f399)
![2024-08-13_101722](https://github.com/user-attachments/assets/03523981-b5d4-402c-a836-8e1cbbdc31dd)

nerdCopter commented:
:( fails for new `XUbuntu 24.04` VM
![image](https://github.com/user-attachments/assets/6bece139-b01a-48ea-9cbc-95af3426337e)
